snippets/csharp/VS_Snippets_CLR/RegularExpressions.Examples.ChangeDateFormats/cs/Example_ChangeDateFormats1.cs

```diff
@@ -18,7 +18,7 @@ static string MDYToDMY(string input)
    {
       try {
          return Regex.Replace(input, 
-               "\\b(?<month>\\d{1,2})/(?<day>\\d{1,2})/(?<year>\\d{2,4})\\b",
+                @"\b(?<month>\d{1,2})/(?<day>\d{1,2})/(?<year>\d{2,4})\b"
```

Comment:
The change for the `using` statement looks great. For this PR, we don't want this change included, as it's in another PR.

Can you revert this commit on this branch?